### ISSUE TITLE
Access rights to the repository

### ISSUE THREAD

**Issue by qligier** [i4mi/MobileAccessGateway#96]
Could I get access rights for creating new branches in this repository please?
Thanks!

**Comment by oliveregger** [comment]
@Alexander-Kreutz could you do that? I'm not able to do it from may side

**Comment by Alexander-Kreutz** [comment]
I cannot do it either. Maybe Francois can?

**Comment by oliveregger** [comment]
@vnf1 bist du admin von dem repo?

**Comment by i4mi** [comment]
@oliveregger: ja, indirekt über den I4MI account. Ich habe Quentin berechtig als "Collaborator".
Hum, es gibt keine differenzierte Rollen wie bei GitLab, z.B. "Owner"...

**Comment by i4mi** [comment]
@qligier, please close the issue when you have successfully accesses

**Comment by qligier** [comment]
Thanks, it seems to work properly!
